test/local-hub/input_validation_test.go

```diff
@@ -180,17 +180,17 @@ func TestExternalStorageInputValidation(t *testing.T) {
 			storageSpec: storageSpec{
 				URL:         ptr.String("http://localhost:5000/fake"),
 				AcceptValue: ptr.Bool(false),
-				ContextSchema: ptr.String(heredoc.Doc(`
+				ContextSchema: []byte(heredoc.Doc(`
```

Comment:
Shouldn't be here `unmarshalContextSchema` as well?

Reply:
Actually, we want to test some incorrect data, which is not even an object, and verify what would be a response from the hub-js so that's why there is a `[]byte`. :)